desktop/cmp/form/FormField.js

```diff
@@ -306,9 +312,19 @@ const editableChild = hoistCmp.factory({
 const blockChildren = ['CodeInput', 'JsonInput', 'Select', 'TextInput'];
 
 function getValidChild(children) {
+    const count = Children.count(children);
+    if (count === 0) return null;
+    if (count > 1) {
+        throw XH.exception('Add a single HoistInput child to FormField, or no children and a readonlyRenderer.');
```

Comment:
Not major, but is this error message leaving out some other options?   The single child is no longer required to be a HoistInput, and if no child is present, is a readOnlyRenderer really required?   A readOnlyRenderer is not a requirement right? because if not present the field will fall back to: defaultReadonlyRenderer right?

Maybe: 'Add a single input child to FormField, or if no children you may specify a custom readonlyRenderer'.

Reply:
great point!